pwnlib/gdb.py

```diff
@@ -464,9 +464,15 @@ def debug(args, gdbscript=None, exe=None, ssh=None, env=None, sysroot=None, **kw
 
     # gdbserver outputs a message when a client connects
     garbage = gdbserver.recvline(timeout=1)
+    garbage += gdbserver.recvline(timeout=1)
```

Comment:
It's also possible that another part of the code just leaves the new line in the tube.